### ISSUE TITLE
[Suggestion] Add raw JSON text format to chatbox tell modes

### ISSUE THREAD

**Issue by hugeblank** [SwitchCraftCC/issues#45]
In addition to the `"markdown"` and  `"format"` I propose that a `"raw"`, or `"json"` mode be added which adds the [`/tellraw` JSON format](https://minecraft.fandom.com/wiki/Raw_JSON_text_format). This mode would obviously retain the prefix and information on whose chatbox the message belongs to. I don't think it's too unfair to ask, since all I think it adds over the other two modes is hover text, command execution (pagination in a chatbox!), and 'obfuscated text'.

Such change would allow for something like [allium-cc](https://github.com/hugeblank/allium-cc) to easily jump from being a creative exclusive utility to potentially survival friendly.

**Comment by emmachase** [comment]
Immediate thoughts: command execution sounds dangerous!

**Comment by hugeblank** [comment]
Yes, but consider that the user had to execute a command, and click on the text that executes the command in the first place. I think that's a lot less harmful than say... providing the ability for a user to bulk spam users with a chatbox in the first place.
